README.md

```diff
@@ -21,6 +21,81 @@ undirected graphs, and electrical circuits.
 * Create an integration with markdown converters, where the visualized
   models can be used in Latex environments and on static web pages.
 
+
+
+### Getting Started
+
+###### **MENTION!** 
+This does not work with Safari as a browser, please test it with a different browser for the moment (Chrome and Firefox work)
+
+
+Before you can start visualizing your models inside a web browser, you need to
+install the following Node packages using NPM (Node Package Manager): http-server
+
+	npm install -g browser-sync
```

Comment:
You should update the documentation, to just install it locally. If so, please also add the relevant node files to .gitignore.